### DISCUSSION TITLE
Clarify file format

### DISCUSSION THREAD

**Discussion by jamjr** [josxha/flutter_map_cache#58] [General]
Maybe i am the only one with this problem, but I think that this library,  that uses the propietary Mapbox pbf format and not the open osm pbf format shoud explicity say that in the docs.  This is confusing and can lead to issues. You can find many "file.osm.pbf" maps out there, but the Mapbox format (mvt) is a paid one.

**Comment by josxha** [comment]
Thanks for bringing up this discussion @jamjr. This is indeed a common misconception. A small correction on your text:

The abbriviation PBF stands for "Protocolbuffer Binary Format" and is used accross different technologies. For example gRPC APIs use Protocol Buffers to minimize the amount of transfered data.

Then there is the usage of PBF encoded data used in mapping:
- The file extension for raw compressed OpenStreetMap data is actually `.osm.pbf` (not just `.pbf`) where `.osm` is the uncompressed OpenStreetMap XML data. You'll have to generate tiles from its data before you are able to use it on a map at a larger scale.
- Mapbox uses for its tiles the suffix `.mvt`, not `.pbf`.
- Other tile servers often use an open source fork of the Mapbox Vector Tiles specification from before Mapbox moved away from their permissive licensing. Those tile providers use the `.pbf` or no suffix on their tile urls.

The plugin supports both, Mapbox tile server and alternative or self hosted tile servers.

You are welcome to open a pull request to improve the documentation for other users.👍
